torch/_inductor/ir.py

```diff
@@ -4423,6 +4426,17 @@ class ComputedBuffer(OperationBuffer):
     """
 
     data: Loops
+    _force_load: ClassVar[bool] = False
+
+    @staticmethod
+    @contextlib.contextmanager
+    def force_load() -> Iterator[None]:
```

Comment:
@karthickai Can we name this `force_realize` instead actually, I think that makes the intent a little clearer. Other than that, once Elias is good w/ this it's good to go